modules/_labs/menu/react/lib/Menu.tsx

```diff
@@ -40,6 +40,11 @@ export interface MenuProps extends GrowthBehavior, React.HTMLAttributes<HTMLULis
    * The HTML `id` of the element that labels the Menu. Often used with menu buttons.
    */
   labeledBy?: string;
+  /**
+   * The role of the Menu. Use this to override the role of the item (e.g. you can use this element as an option in a Combobox).
+   * @default menu
+   */
+  role: string;
```

Comment:
Why does this bug fix involve expanding the API? I can't imagine a menu component having any other role than `menu`

Reply:
i can revert this, but i saw that the role in menu wasn't a prop

Reply:
The only reason it's configurable for `MenuItem` is because they can also be used in a combobox (who's items require a different role). I think we're good to keep `Menu`'s role static 👍